src/core/messagelogmodel.h

```diff
@@ -77,7 +77,7 @@ class MessageLogModel : public QAbstractListModel
   private:
     QgsMessageLog *mMessageLog = nullptr;
     QVector<LogMessage> mMessages;
-    QList<QString> mSuppressedTags;
+    QMap<QString, QStringList> mSuppressedModel;
```

Comment:
Model seems off here -- 
```suggestion
    QMap<QString, QStringList> mSuppressedFilters;
```